### DISCUSSION TITLE
More info on badges/achievments

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46185] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I think there should be a way to view all possible badges/achievements (like a catalog) and how to earn each badge/achievement.

The above 'n' beyond feature would be to see who gets each achievement/badge.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @eggnaut 

While there is no such feature, you can refer to this link:

https://github.com/drknzz/GitHub-Achievements

**Reply by ghost** [reply]
Thank you, that was really helpful!

My main reason for posting here is because I clicked 'give feedback' on the Github badges.

**Reply by WahomeKezia** [reply]
> Hey @eggnaut
> 
> While there is no such feature, you can refer to this link:
> 
> https://github.com/drknzz/GitHub-Achievements

This is really helpful. Best answer I have seen on the topic of badges! @SilasRodrigues19 🙌🙌
